web-api/storage/scripts/cases/createOrder.js

```diff
@@ -52,6 +52,7 @@ module.exports.createOrder = async ({ docketNumber }) => {
         .serveCourtIssuedDocumentInteractor(applicationContext, {
           docketEntryId,
           docketNumber,
+          subjectCaseDocketNumber: docketNumber,
```

Comment:
this is something extra we found related to 9513 https://github.com/ustaxcourt/ef-cms/pull/2383